README.md

```diff
@@ -47,6 +47,30 @@ It's same as `-level` flag of reviewdog.
 Optional. Reporter of reviewdog command [`github-pr-check`, `github-pr-review`].
 The default is `github-pr-check`.
 
+### `rubocop_version`
+
+**Required**. Set rubocop version. 
```

Comment:
I would like these flags to be optional. I mean if a user doesn't specify the version, we will install the latest version